services/QuillLMS/app/views/pages/home_new.html.erb

```diff
@@ -28,7 +30,8 @@
     <div class='info-section'>
       <a href="/tools/connect/"><h2 class='q-h2'><img class="icon-white lazyload" data-src='https://assets.quill.org/images/shared/quill_connect_white.svg'>Quill Connect</h2></a>
       <p class='blurb'>Help your students advance from fragmented and run-on sentences to complex and well structured ones.</p>
-        <div class="light-bar"></div>
+      <%= render_dash(@is_mobile) %>
```

Comment:
I'm noticing that `render_dash` always precedes `render_content` in this PR.  If you think it's likely this will be the case in the future, you could make that coupling explicit:

```ruby
def render_content(device, type, url)
  render_dash(device)
  ...
end
```
This would also eliminate the `@is_mobile` instance variable.